pkg/controller/ingressconfig/ingressconfig_controller.go

```diff
@@ -142,5 +142,7 @@ func (r *ReconcileIngressConfigs) updatePolicyGroupLabelOnNamespace(ctx context.
 
 	newNamespace.SetLabels(existingLabels)
 
-	return r.client.Patch(context.TODO(), newNamespace, crclient.MergeFrom(namespace))
+	return r.client.Patch(context.TODO(), newNamespace, crclient.MergeFrom(namespace), &crclient.PatchOptions{
```

Comment:
@npinaeva @kyrtapz This seems to be using MergePatchType, shouldn't this be changed to use ApplyPatchType for SSA to kick in ?